dialogue-core/build.gradle

```diff
@@ -2,8 +2,6 @@ apply from: "$rootDir/gradle/publish-jar.gradle"
 apply plugin: 'com.palantir.revapi'
 apply plugin: 'com.palantir.metric-schema'
 
-sourceCompatibility = 11
```

Comment:
you might want to set `sourceCompatibility = 1.8` explicitly to avoid an accidental regression

Reply:
Should we do that in all the modules? The other modules which are jdk8-compatible inherit sourceCompatibility from the root build.gradle.